presto-spi/src/main/java/com/facebook/presto/spi/SystemTable.java

```diff
@@ -29,9 +29,27 @@
 
     /**
      * Create a cursor for the data in this table.
+     * <p>
+     * Derived class need to implement one of cursor and pageSource.
```

Comment:
Phrase this as `Exactly one of {@link #cursor} or {@link #pageSource} must be implemented.`